### PR TITLE
Prepare for release 1.0.0-rc.2

### PR DESCRIPTION
**Pull request by mateuszrzeszutek**

### CODE REVIEW

**Review by theletterf — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,20 @@ adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
...
+Release candidate for 1.0.0.
+
+* Updated to OpenTelemetry Java v1.19.0 (#383)
+* Fixed a bug where sometimes crash reports were not exported (#368)
+* Added runtime details (storage, memory, battery) to crash reports (#369)
+* Removed deprecated classes (#372)
+* Included activity name in slow and frozen render reports (#373)
+* Renamed the `SplunkRumBuilder#disableNetworkMonitorEnabled()` method to
+  `SplunkRumBuilder#disableNetworkMonitor()` (#377)
+* Added experimental React Native support (React Native lib will be developed and weleased
```
Comment (breedx-splk) [inline CHANGELOG.md:23]:
```suggestion
* Added experimental React Native support (React Native lib will be developed and released
```

File: splunk-otel-android/src/main/java/com/splunk/rum/SplunkRumBuilder.java [hunk 2]
```diff
@@ -113,8 +113,7 @@ public SplunkRumBuilder setRumAccessToken(String rumAuthToken) {
     /**
      * Enables debugging information to be emitted from the RUM library.
      *
-     * <p>This feature is disabled by default. You can enable it by calling this configuration
-     * method with a {@code true} value.
+     * <p>This feature is disabled by default. You can enable it by calling configuration method.
```
Comment (breedx-splk) [inline splunk-otel-android/src/main/java/com/splunk/rum/SplunkRumBuilder.java:116]:
```suggestion
     * <p>This feature is disabled by default. Call this method to enable debugging information to be emitted.
```

Comment (breedx-splk) [inline splunk-otel-android/src/main/java/com/splunk/rum/SplunkRumBuilder.java:116]: Alternately (for parity with the other javadocs):

```suggestion
     * <p>This feature is disabled by default. You can enable it by calling this method.
```

**Review by breedx-splk — APPROVED** [review]
Sweet, looking good! A few small small typo/grammer things.

File: CHANGELOG.md (hunk 1 above)
Comment (theletterf) [reply CHANGELOG.md:23]: Weasel + released. There, I said it.

**Pull request merged**